pkg/apis/velero/v1/backup_types.go

```diff
@@ -159,6 +160,9 @@ type BackupSpec struct {
 	// The default value is 1 hour.
 	// +optional
 	ItemOperationTimeout metav1.Duration `json:"itemOperationTimeout,omitempty"`
+	// ResourcePolicies specifies the referenced resource policies that backup should follow
+	// +optional
+	ResourcePolicies *v1.TypedLocalObjectReference `json:"resourcePolices,omitempty"`
```

Comment:
Let's call it `ResourcePolicy`. Usually, complex means an object list. Change the command option as well

Reply:
updated